precompile_process_test.go

```diff
@@ -59,6 +75,28 @@ func testPrecompileProcess(t *testing.T, context spec.G, it spec.S) {
 			Expect(executions[0].Env).To(ContainElement("SECRET_KEY_BASE=dummy"))
 		})
 
+		it("runs the bundle exec assets:precompile process while respecting RAILS_ENV", func() {
+			os.Setenv("RAILS_ENV", "staging")
+			err := precompileProcess.Execute(workingDir)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(executions).To(HaveLen(1))
+			Expect(executions[0].Args).To(Equal([]string{"exec", "rails", "assets:precompile", "assets:clean"}))
+			Expect(executions[0].Env).To(ContainElement("RAILS_ENV=staging"))
+			Expect(executions[0].Env).To(ContainElement("SECRET_KEY_BASE=dummy"))
+		})
+
+		it("runs the bundle exec assets:precompile process while respecting SECRET_KEY_BASE", func() {
+			os.Setenv("SECRET_KEY_BASE", "dummy2")
+			err := precompileProcess.Execute(workingDir)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(executions).To(HaveLen(1))
+			Expect(executions[0].Args).To(Equal([]string{"exec", "rails", "assets:precompile", "assets:clean"}))
+			Expect(executions[0].Env).To(ContainElement("RAILS_ENV=production"))
```

Comment:
The unit tests appear to be failing on this line, due to `RAILS_ENV` being set to `staging` if you're able to fix that